### ISSUE TITLE
Qwen-7b int8 inference speed is too slow, only 3-4 tokens/s.

### ISSUE THREAD

**Issue by Liqiandi** [intel-analytics/ipex-llm#11289]
I used the code below to test the speed of ipex-llm generating tokens, but the result was only 3-4tokens/s.

`

def main(model_dir = "Qwen/Qwen-7B-Chat"):

    seed = 1024
    max_experiment_times = 1
    context_length_per_experiment = 1
    generate_length_per_experiment = 2048
    
    use_flash_attn = False
    set_seed(seed)

    tokenizer = AutoTokenizer.from_pretrained(model_dir, trust_remote_code=True)
    model = AutoModelForCausalLM.from_pretrained(model_dir,trust_remote_code=True)
    model = optimize_model(model, low_bit = 'sym_int8').to('xpu')
  
    print("模型参数以及配置加载完毕\n")

    time_costs = []
    context_str = '我' * context_length_per_experiment
    max_gpu_memory_cost = 0
    for _ in tqdm(range(max_experiment_times)):
        with torch.no_grad():
            input_ids = tokenizer.encode(context_str, return_tensors="pt").to('xpu')
        print("tokenizer执行结束\n")
        t1 = time.time()
            
        pred = model.generate(input_ids, min_length=generate_length_per_experiment+context_length_per_experiment, max_new_tokens=generate_length_per_experiment)
        output_str = tokenizer.decode(pred[0], skip_special_tokens=True)
        print(output_str)
        time_costs.append(time.time() - t1)
        max_gpu_memory_cost = max(max_gpu_memory_cost, torch.cuda.max_memory_allocated())
        torch.cuda.empty_cache()         
    
    print(model_dir) 
    print(f"time_costs = {sum(time_costs)}")
    # print("Average generate speed (tokens/s): {}".format((max_experiment_times * generate_length_per_experiment) / sum(time_costs)))
    print("Average generate speed (tokens/s): {}".format((max_experiment_times * pred.shape[1]) / sum(time_costs)))
    print(f"GPU Memory cost: {max_gpu_memory_cost / 1024 / 1024 / 1024}GB")
    print("Experiment setting: ")
    print(f"seed = {seed}")
    print(f"max_experiment_times = {max_experiment_times}")
    print(f"context_length_per_experiment = {context_length_per_experiment}")
    print(f"generate_length_per_experiment = {generate_length_per_experiment}")
    print(f"use_flash_attn = {use_flash_attn}")
    # print(f"quant_type = {quant_type}")
    print("\n")

`

Of course, this code does not take into account the warm up time required to generate the first tokens. However, when I use another method for infering like the below code, it takes more than ten seconds to generate about 20 words each time, which is obviously too slow. Yes, I would like to ask if there is something wrong with the way I load the model?
`

 messages = [
            {"role": "system", "content": "some prompts”"},
            {"role": "user", "content": question}
            ]
        text = self.tokenizer.apply_chat_template(
            messages,
            tokenize=False,
            add_generation_prompt=True
            )
        st = time.time()
        model_inputs = self.tokenizer([text], return_tensors="pt").to('xpu')
        
        generated_ids = self.model.generate(
            model_inputs.input_ids,
            max_new_tokens=self.n_predict
            )
        end = time.time()
        generated_ids = [
            output_ids[len(input_ids):] for input_ids, output_ids in zip(model_inputs.input_ids, generated_ids)
            ]

        response = self.tokenizer.batch_decode(generated_ids, skip_special_tokens=True)[0]
        print(f'Part---Inference time: {end-st} s')

`

**Comment by jason-dai** [comment]
@Liqiandi please follow https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Quickstart/benchmark_quickstart.html for benchmark ipex-llm performance
